jdaviz/configs/default/plugins/model_fitting/model_fitting.py

```diff
@@ -947,17 +947,9 @@ def _fit_model_to_cube(self, add_data):
                 temp_label = "{} ({}, {})".format(self.results_label, m["x"], m["y"])
                 self.app.fitted_models[temp_label] = m["model"]
 
-        count = max(map(lambda s: int(next(iter(re.findall(r"\d$", s)), 0)),
-                        self.data_collection.labels)) + 1
-
-        label = self.app.return_data_label(f"{self.results_label}[Cube]", ext=count)
```

Comment:
I am not sure what this label is for because `self.add_results.add_results_from_plugin` has its own label stuff going on in the mixin.

Reply:
@javerbukh - any thoughts?

Reply:
No thoughts mean okay to remove?

Reply:
Yeah I don't think that gets used so all good to remove it.